cpp/src/cylon/join/join.cpp

```diff
@@ -44,6 +45,28 @@ inline void advance(std::vector<int64_t> *subset,
       break;
     }
     data_index = sorted_indices->Value(*current_index);
+  }MultiTableRowIndexComparator
+}
+
+inline void advance2(std::vector<int64_t> *subset,
```

Comment:
shall we rename this to advance_multi_index or something?

Reply:
This function doesn't even belong to this file anymore.. And it's already renamed.. I guess I have something that hasn't pushed to the remote. This is not what I have locally.. :-D